recipes/sparc-x-api/meta.yaml

```diff
@@ -8,20 +8,19 @@ package:
 
 source:
   url: https://github.com/SPARC-X/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 34aa3a8d0801a91736b0d8322d78d1e432e92585e61af54db3aba17340ff7617
+  sha256: dca059a2bc4a020050ce8983b247176471eeb76e4a4dd96708133e0fcad18e19
 
 build:
   noarch: python
   number: 0
   # Install and include the psp files
-  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv && cd .. && {{ PYTHON }} -m sparc.download_data"
   entry_points:
     - "sparc-ase = sparc.cli:main"
 
 requirements:
   host:
     - python >=3.8
-    - pip
```

Comment:
You do need `pip` here b/c we use it in the python pip install command above.

Reply:
@ocefpaf Sorry for that. I've updated the meta.yaml.